.github/workflows/python.yml

```
@@ -75,7 +75,7 @@ jobs:
     timeout-minutes: 45
```

Would you be willing to add a label to the runs so it's clearer what the numbers mean in the run title?
```suggestion
    run-name: Linux (Python 3.{{ matrix.python-minor-version }})
    timeout-minutes: 45
```

---

done